sdk/core/core/src/az_write_span_iter.c

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_write_span_iter.h>
+
+#include <_az_cfg.h>
+
+az_result az_write_span_iter_write(az_write_span_iter * const p_i, az_const_span const span) {
```

Comment:
"write span iter write" seems like a mouthful.  Can this be shortened?

Reply:
yes, I would like to rename to `span_builder` type and `span_builder_apeend`. Issue #88